src/features/events/components/EventActionButtons.tsx

```diff
@@ -17,6 +18,7 @@ import { ZetkinEvent } from 'utils/types/zetkin';
 import { ZUIConfirmDialogContext } from 'zui/ZUIConfirmDialogProvider';
 import ZUIDatePicker from 'zui/ZUIDatePicker';
 import ZUIEllipsisMenu from 'zui/ZUIEllipsisMenu';
+import EventMoveDialog from './EventChangeCampaignDialog';
```

Comment:
The name of the import should change to reflect the name of the export and file.